src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -43,6 +43,26 @@ describe('ReactDOMFiber', () => {
     expect(container.textContent).toEqual('10');
   });
 
+  it('should be called a callback argument', () => {
```

Comment:
@gaearon Thanks! I've fixed that `this` in render callbacks is the public instance.
Then I confirmed the tests linked from your comment are passed when `unstable_batchedUpdates` is mocked.

I'll fix it after this PR was merged.